packages/i18n/README.md

```diff
@@ -181,15 +170,7 @@ console.log(t('hello', ['World'])); // Hello World
 
 ### Locale setting on development
 
-If you want to show specific language, you can set the `devLocale` property. (only for development)
-
-```typescript
-import { t } from '@extension/i18n';
-
-t.devLocale = "ko";
-
-console.log(t('hello')); // 안녕
-```
+If you want to enforce displaying specific language, you need to set `CEB_DEV_LOCALE` in `.env` file (work only for development).
```

Comment:
`CEB_DEV_LOCALE` env is not working yet. right?

Reply:
Yeah, it was placeholder till I'll merge #725 
That's why i've changed branch to `dev`